src/features/error-highlighting.ts

```diff
@@ -128,10 +128,12 @@ const Elm = {
     const promise: Promise<ParsedError | undefined> =
       new Promise((resolve, reject) => {
         const isWindows = process.platform === 'win32'
+        const compiler = vscode.workspace.getConfiguration('elmLand').compilerFilepath;
+        console.log('compilerFilepath:', compiler);
```

Comment:
Not bullish on this either, figured maybe the visibility would be helpful if debugging, but happy to remove it.